### PR TITLE
:sparkles: Add helpful account related hints in record preview card

### PR DESCRIPTION
**Pull request by foysalit**
Oftentimes, when reviewing a piece of record/content in quick action panel, some account metadata/hint can aid review process a lot. For instance, if a record being reviewed is from a new account or deactivated account can make a lot of difference in the outcome of the review.

This PR adds a few chip elements containing account hints for 1. deactivated 2. takendown and 3. new accounts, right under the preview card when the subject being reviewed is not an account, 

Screenshot 2024-09-03 at 00 21 06

https://github.com/user-attachments/assets/14b3ffa0-aad3-4440-bd46-2a6509b5ec63

### CODE REVIEW

File: app/actions/ModActionPanel/QuickAction.tsx [hunk 1]
```diff
@@ -565,22 +546,7 @@ function Form(
...
-                        : 'reviewed'}{' '}
-                      at:{' '}
-                      {dateFormatter.format(
-                        new Date(subjectStatus.lastReviewedAt),
-                      )}
-                    </span>
-                  ) : (
-                    <span className="pl-1">Not yet reviewed</span>
-                  )}
+                  <LastReviewedTimestamp subjectStatus={subjectStatus} />
```
Comment (foysalit) [inline app/actions/ModActionPanel/QuickAction.tsx:549]: Unrelated refactor.

File: components/mod-event/constants.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const MOD_EVENT_TITLES = {
   [MOD_EVENTS.EMAIL]: 'Email Sent',
   [MOD_EVENTS.RESOLVE_APPEAL]: 'Appeal Resolved',
   [MOD_EVENTS.TAG]: 'Tag',
+  [MOD_EVENTS.APPEAL]: 'Appeal',
```
Comment (foysalit) [inline components/mod-event/constants.ts:39]: Unrelated fix.

File: components/subject/RecordAuthorStatus.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      })
+    : ''
+
+  const isTakendown = !!repo.moderation.subjectStatus?.takendown
+  return (
+    <>
+      {(isNew || isYoung) && (
+        <LabelChip
+          className="dark:bg-slate-700 dark:text-gray-200 rounded-xs"
+          title={`Account was created less than ${NEW_ACCOUNT_MARKER_THRESHOLD_IN_DAYS} days ago`}
```
Comment (matthieusieben) [inline components/subject/RecordAuthorStatus.tsx:35]: Shouldn't the `YOUNG_ACCOUNT_MARKER_THRESHOLD_IN_DAYS` be used instead of `NEW_ACCOUNT_MARKER_THRESHOLD_IN_DAYS` when `isYoung` is true but not `isNew`?

**Review by matthieusieben — APPROVED** [review]
looks good

File: components/common/feeds/RecordCard.tsx [hunk 4]
```diff
@@ -51,7 +51,7 @@ export const FeedGeneratorRecordCard = ({ uri }: { uri: string }) => {
   }
 
   return (
-    <div className="bg-white">
+    <div className="bg-white dark:bg-slate-800">
```
Comment (foysalit) [inline components/common/feeds/RecordCard.tsx:54]: another unrelated thing i just happened to notice while debugging something.
